### DISCUSSION TITLE
AS Information

### DISCUSSION THREAD

**Discussion by tonniteles** [phaag/nfdump#519] [General]
Hi Peter, 
As you requested, I opened a new ticket.
I made the updates, but nfdump did not return the AS number.
I even used your "trick", but it didn't work either.

`<
root@flow-testes:/# nfdump -V
nfdump: Version: 1.7.4-a16f86f Options: NSEL-NEL Date: 2024-04-07 15:15:55 +0200

root@flow-testes:~# nfdump -G mmdb.nf -r /var/spool/flows/2024/04/08/14/nfcapd.202404081405 -s dstas
Top 10 Dst AS ordered by flows:
Date first seen             Duration     Proto                Dst AS    Flows(%)     Packets(%)       Bytes(%)         pps      bps   bpp
Summary: total flows: 216957, total bytes: 18.0 G, total packets: 19.1 M, avg bps: 78.2 M, avg pps: 10392, avg bpp: 940
Time window: 2024-04-06 22:37:57 - 2024-04-06 23:08:34
Total flows processed: 216957, passed: 216957, Blocks skipped: 0, Bytes read: 36761828
Sys: 0.6117s User: 1.3329s Wall: 0.0475s flows/second: 4571564.4 Runtime: 0.0477s

 nfdump -G mmdb.nf -r /var/spool/flows/2024/04/08/14/nfcapd.202404081405 -o csv 'not src as 65535' > flow.csv
{
        "type" : "FLOW",
        "export_sysid" : 1,
        "first" : "2024-04-06T23:07:54.950",
        "last" : "2024-04-06T23:07:54.950",
        "received" : "2024-04-08T14:09:20.745",
        "in_packets" : 1,
        "in_bytes" : 199,
        "proto" : 17,
        "tcp_flags" : "........",
        "src_port" : 53,
        "dst_port" : 32880,
        "fwd_status" : 0,
        "src_tos" : 0,
        "src4_addr" : "8.8.4.4",
        "dst4_addr" : "192.141.190.139",
        "src4_geo" : "NA/US/EUA long/lat: 37.7510/-97.8220",
        "dst4_geo" : "SA/BR/"Rio de Janeiro" long/lat: -22.4549/-41.9531",
        "input_snmp" : 27,
        "output_snmp" : 25,
        "src_mask" : 0,
        "dst_mask" : 0,
        "src_net" : "",
        "dst_net" : "",
        "direction" : 0,
        "dst_tos" : 0,
        "ip4_next_hop" : "192.141.190.2",
        "ip4_router" : "192.141.190.1",
        "in_src_mac" : "60:12:3c:79:8a:b0",
        "out_dst_mac" : "00:00:00:00:00:00",
        "in_dst_mac" : "b8:69:f4:11:24:63",
        "out_src_mac" : "b8:69:f4:11:24:63",
        "src4_xlt_ip" : "8.8.4.4",
        "dst4_xlt_ip" : "100.64.1.64",
        "src_xlt_port" : 53,
        "dst_xlt_port" : 32880,
        "sampled" : 0
},

>`

**Comment by phaag** [comment]
I have created a new issue.

**Comment by phaag** [comment]
Fixed in master branch
